src/app/create-profile/page.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import { UUID } from 'crypto';
+import { useState } from 'react';
+import { insertProfile } from '../../api/supabase/queries/profiles';
+import { Profile } from '../../types/schemaTypes';
+import styles from '../page.module.css';
+
+export default function Profile() {
+  const [firstName, setFirstName] = useState<string>('');
+  const [lastName, setLastName] = useState<string>('');
+
+  const handleInsert = async () => {
+    if (!firstName && !lastName) return;
```

Comment:
Are we allowing users to not input a first name or a last name? This line allows user to have either first or last (rather than requiring them to have both). Change to `!firstName || !lastName` if we want to require both first and last?

Reply:
ugh such good feedback THANKS!!